src/itemLogin/ItemLoginAuthorization.tsx

```diff
@@ -21,8 +21,8 @@ export type ItemLoginAuthorizationProps = {
   signIn: (args: { itemId: string } & SignInPropertiesType) => void;
   itemId: UUID;
   useCurrentMember: () => UseQueryResult<MemberRecord>;
-  useItem: (itemId: string) => UseQueryResult<ItemRecord>;
-  useItemLogin: (itemId: string) => UseQueryResult<ItemLoginRecord>;
+  useItem: (itemId?: string) => UseQueryResult<ItemRecord>;
+  useItemLogin: (itemId?: string) => UseQueryResult<ItemLoginRecord>;
```

Comment:
I guess this will simply return or error out if they are undefined. But should we enforce this?

Reply:
This typing is more consistent with the typing of the query. But you are right the behavior of the query is to error with "Undefined Argument" when the value is `undefined`.